api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -385,7 +503,7 @@ public CohortDefinition getPatientsWhoWereActiveByEndOfPreviousMonthB12() {
     CohortDefinition transferredOut = getPatientsTransferredOut();
     CohortDefinition suspended = getPatientsWhoSuspendedTreatment();
     CohortDefinition missedDrugPickup = getLastArvPickupDateCohort();
-    CohortDefinition died = getPatientsWhoDied();
+    CohortDefinition died = getPatientsWhoDied(false);
```

Comment:
Since you are already working on `B12`, this change if fine. If it was someone else working on it  in parallel this could cause problems. We need a better way to plan this changes.

Reply:
Will pay attention next time, is adding comments a viable alternative?